docs/v3_services.rst

```diff
@@ -0,0 +1,237 @@
+Workstation* with ``./securedrop-admin tailsconfig``.
+
+- First, boot into the *Admin Workstation* with the persistent volume unlocked
+  and an admin password set.
+- Next, open a terminal via **Applications > Favorites > Terminal**.
+- Verify that SecureDrop version 1.0.0 or greater is available or installed on
+  your instance with the command:
+
+  .. code:: sh
+
+    ssh app apt-cache policy securedrop-app-code
+
+  Version 1.0.0 should be listed as installed or as an installation candidate.
+- Verify that the *Admin Workstation*'s SecureDrop code is on 1.0.0 or greater,
+  using the GUI updater or the command:
+
+  .. code:: sh
+ 
+    ./securedrop-admin update
+
+- Next, enable v3 onion services (and optionally disable v2 services) using:
+
+  .. code:: sh
+    
+    ./securedrop-admin sdconfig
+
+  This command will step through the current instance configuration. None of the
+  current settings should be changed. When prompted to enable v2 and v3 
+  services, you should choose either ``yes`` to both to use v2 and v3 
+  concurrently, or ``no`` to v2 and ``yes`` to v3 to migrate to v3 only. 
+
+- Once the configuration has been updated, run the installation playbook using 
+  the command:
+
+  .. code:: sh
+  
+    ./securedrop-admin install
+
+  This will enable v3 onion services on the *Application* and *Monitor Servers*.
+
+- When the installation playbook run is complete, update the *Admin Workstation*
+  to use v3 onion services using the command:
+
+  .. code:: sh
+  
+    ./securedrop-admin tailsconfig
+
+- Next, verify connectivity between the *Admin Workstation* and the SecureDrop
+  instance as follows:
+
+  - Use the Source desktop shortcut to connect to the *Source Interface* and 
+    verify that the new 56-character address is present in the Tor Browser 
+    address bar
+  - Use the Journalist desktop shortcut to connect to the *Journalist Interface*
+    and verify that the new 56-character address is present in the Tor Browser 
+    address bar
+  - Use the commands ``ssh app`` and ``ssh mon`` in a terminal to verify 
+    SSH access to the *Application* and *Monitor Servers*.
+
+- Finally, back up the instance and *Admin Workstation* USB.
+
+(Optional) enabling HTTPS
+^^^^^^^^^^^^^^^^^^^^^^^^^
+If your instance serves the *Source Interface* over HTTPS, and you plan to 
+continue using HTTPS with v3 onion services, you'll need to provision a 
+new certificate for the new v3 address.
+
+You'll find the new *Source Interface* address in the file:
+
+.. code-block:: none
+ 
+  ~/Persistent/securedrop/install_files/ansible-base/app-sourcev3-ths
+
+Follow the instructions in :doc:`HTTPS on the Source Interface <https_source_interface>`
+to provision and install the new certificate. Note that you will not be able to
+provide HTTPS versions of both the v2 and v3 *Source Interface* - it may be best
+to set up v3 HTTPS as a last step before publicising the new address.
```

Comment:
> Note that you will not be able to provide HTTPS versions of both the v2 and v3 *Source Interface*

Is that the final word? I seem to recall Jen mentioning something about finding an example of an EV cert that included both a v2 and a v3 address.

Reply:
([see here](https://github.com/freedomofpress/securedrop/issues/4677#issuecomment-523986821) where I found some certs issued for both v2 and v3 onions)

Reply:
Nice, deleted.